pkg/agent/api/delegatedidentity/v1/service.go

```diff
@@ -120,6 +130,11 @@ func (s *Service) SubscribeToX509SVIDs(req *delegatedidentityv1.SubscribeToX509S
 		return status.Error(codes.InvalidArgument, "could not parse provided selectors")
 	}
 
+	log.WithFields(logrus.Fields{
+		"delegate_selectors": cachedSelectors,
+		"request_selectors":  selectors,
+	}).Debug("Subscribing to cache changes")
```

Comment:
```suggestion
	}).Info("Subscribing to cache changes")
```
This seems pretty useful for tracing requests. I think we can bump this to INFO level.